src/ngraph/pass/constant_folding.cpp

```diff
@@ -529,3 +531,76 @@ void ngraph::pass::ConstantFolding::construct_constant_dequantize()
     auto dequantize_matcher = make_shared<pattern::Matcher>(dequant, constant_dequantize_callback);
     this->add_matcher(dequantize_matcher);
 }
+
+template <class REAL, class QUANT>
+shared_ptr<op::Constant> make_constant_quantize(shared_ptr<op::Constant> constant,
+                                                shared_ptr<op::Quantize> quant,
+                                                shared_ptr<op::Constant> scale,
+                                                shared_ptr<op::Constant> offset)
+{
+    auto out_shape = constant->get_shape();
+    vector<QUANT> out_vec(shape_size(out_shape));
+
+    runtime::reference::quantize<REAL, QUANT>(constant->get_vector<REAL>().data(),
+                                              scale->get_vector<REAL>().data(),
+                                              offset->get_vector<QUANT>().data(),
+                                              out_vec.data(),
+                                              constant->get_shape(),
+                                              scale->get_shape(),
+                                              quant->get_axes());
+
+    return make_shared<op::Constant>(quant->get_element_type(), out_shape, out_vec);
+}
+
+void ngraph::pass::ConstantFolding::construct_constant_quantize()
+{
+    auto constant_label =
+        make_shared<pattern::op::Label>(element::f32, Shape{2}, pattern::has_class<op::Constant>());
+    auto q_scale = op::Constant::create(element::f32, Shape{}, {1});
+    auto q_offset = op::Constant::create(element::i8, Shape{}, {1});
```

Comment:
0 seems a better default value for offset.

Reply:
done